Tools/cases_generator/generate_cases.py

```diff
@@ -381,105 +426,211 @@ def check_families(self) -> None:
                         family,
                     )
 
-    def analyze_supers(self) -> None:
-        """Analyze each super instruction."""
+    def analyze_supers_and_macros(self) -> None:
+        """Analyze each super- and macro instruction."""
```

Comment:
We now do supers and macros separately -- the code is actually a bit bulkier (will work on that later) but more straightforward because the subtle differences between one and the other aren't triggered using flags.